developer-portal-wrapper/developer_portal_wrapper.py

```diff
@@ -70,14 +64,17 @@ def generate_metadata(
                 "productCode": product_code,
                 "productVersionName": product_version_name,
                 "downloadURL": generate_download_url(content_dir, file),
+                "shortURL": f"{short_url_prefix}/{file}",
                 "label": file,
+                "type": "FILE",
+                "hidden": False,
+                "invisible": False,
             }
-            default_values_per_component["shortURL"] += f"/{file}"
             metadata.append(
                 {
                     "type": "file",
                     "action": "create",
-                    "metadata": {**default_values_per_component, **component_item},
+                    "metadata": {**component_item},
```

Comment:
I could have said that... I never used anything like this personally, but AFAICS, the ** was used to unpack the dictionaries to be able to combine the two together. But now that it's just one, we can simply do:
```suggestion
                    "metadata": component_item,
```

Reply:
updated. thanks